### PR TITLE
Add Stack Overflow company blurb to activities/recruitment

### PR DESCRIPTION
**Pull request by ElenaFdR**
So that I can share it with potential applicants for the [communications editor/storyteller](https://publiccode.net/careers/content-editor.html) role.

-----
[View rendered activities/recruitment/company-description.md](https://github.com/publiccodenet/about/blob/add-Stack-Overflow-company-description/activities/recruitment/company-description.md)
[View rendered activities/recruitment/index.md](https://github.com/publiccodenet/about/blob/add-Stack-Overflow-company-description/activities/recruitment/index.md)

### CODE REVIEW

File: activities/recruitment/company-description.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+* Find transformative codebases to reuse
+* Design their code in a way that encourages international reuse
+* Build thriving open source communities around their codebases
+* Get institutional support in navigating the risk and procurement hurdles to using open source for public services and infrastructure
+
+We do this through active codebase stewardship and by publishing the [Standard for Public Code](https://standard.publiccode.net/).
+
+### How we work
+
+All of us fundamentally believe that our public organizations using FOSS will result in better public services for everyone and be a better use of taxpayer money. We do impactful work as part of a mission we believe in.
```
Comment (Ainali) [inline activities/recruitment/company-description.md:24]: I guess FOSS might work on Stack Overflow, but for other places we should perhaps not use the abbreviation unexplained.

File: activities/recruitment/company-description.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+We hire the best people we can, trust their expertise, and give them near infinite autonomy. We also support each other, and do lots of pair working.
+
+Because nobody has ever done this before for government and we've just started, everything we're doing is greenfield - everyone here comes to work every day knowing that what they'll do that day will meaningfully shape the Foundation's future.
+
+We're ambitious but humble - we sincerely want this to succeed, but there's no guarantee. Because we believe that failure is the best teacher, we encourage curiosity, experimentation and learning.
+
+### Benefits
+
+* 25 days of paid leave plus an average of 6 public holidays
+* enthusiastic BYOD culture
```
Comment (Ainali) [inline activities/recruitment/company-description.md:37]: I think BYOD should also be explained to make the text more accessible

File: activities/recruitment/company-description.md (hunk 1 above)
Comment (ElenaFdR) [reply activities/recruitment/company-description.md:24]: Thanks!

File: activities/recruitment/company-description.md (hunk 2 above)
Comment (ElenaFdR) [reply activities/recruitment/company-description.md]: done!

**Review by Ainali — APPROVED** [review]
Yay!

**Review by Ainali — APPROVED** [review]

**Comment by ElenaFdR** [comment]
All links flagged by Travis work in the browser, so I'm merging.

**Pull request merged**